cmd/agent/app/integrations.go

```diff
@@ -404,6 +415,38 @@ func pipCheck(cachePath string) error {
 	return fmt.Errorf("error executing pip check: %v", err)
 }
 
+func moveConfigurationFiles(integration string) error {
+	confFolder := config.Datadog.GetString("confd_path")
+	check := strings.TrimPrefix(integration, "datadog-")
+	confFileDest := filepath.Join(confFolder, fmt.Sprintf("%s.d", check))
+	if err := os.MkdirAll(confFileDest, os.ModeDir); err != nil {
+		return err
+	}
+
+	here, _ := executable.Folder()
+	confFileSrc := filepath.Join(here, relChecksPath, check, "data")
```

Comment:
a couple of notes on things that could break in the future (e.g.: if we change where the python's `site-packages` is for these wheels, if wheels change their structure, etc):

1. _ideally_, `relChecksPath` should be given by pip (`pip show <wheel_name>` gives where the wheel's installed). Not sure we actually want to rely on `pip` though, as parsing pip's output has downsides as well (what if pip's output format changes?)
2. can we rely on the fact that all the files in the wheel's `data` folder should be copied to `conf.d`?